File: 05 ANDROID STUDIO JAVA AUTOMATED LETTER WRITING (PRE- TRAINING PROJECT)/app/src/main/java/com/kenetic/practiceset/MainActivity.java
package com.kenetic.practiceset;

import androidx.appcompat.app.AppCompatActivity;

import android.content.ClipData;
import android.content.ClipboardManager;
import android.content.Context;
import android.os.Bundle;
import android.view.View;
//import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void copyToClip(View view) {

        EditText firstNameET = findViewById(R.id.yourFirstName);
        EditText lastNameET = findViewById(R.id.yourLastName);
        EditText companyNameET = findViewById(R.id.companyName);
        EditText recruiterNameET = findViewById(R.id.recruiterName);
        EditText enterLetterContentET = findViewById(R.id.enterLetterContent);
        //Button copyToClipBT = findViewById(R.id.copyToClipboard);

        String firstName = firstNameET.getText().toString();
        String lastName = lastNameET.getText().toString();
        String companyName = companyNameET.getText().toString();
        String recruiterName = recruiterNameET.getText().toString();
        String enterLetterContent = enterLetterContentET.getText().toString();
        String toCopy = String.format("hello %s,\nmy name is %s %s.\nI have gone through the requirements posted by you and I believe that\nI might be the best fit for the required job post at \n\t%s\nAnd, here is why i think so - \n\n%s",recruiterName,firstName,lastName,companyName,enterLetterContent);

        ClipboardManager clipboard = (ClipboardManager) getSystemService(Context.CLIPBOARD_SERVICE);
        ClipData clip = ClipData.newPlainText("Copied Text",toCopy);
        clipboard.setPrimaryClip(clip);
        Toast.makeText(MainActivity.this,"copied",Toast.LENGTH_SHORT).show();
        //ClipboardManager clip = (ClipboardManager) getSystemService(Context)
    }
}

File: 12 AFFERMATIONS APP (RECYCLER VIEW) (GOOGLE TRAINING)/app/src/main/java/com/kenetic/affermationsapp/MainActivity.kt
package com.kenetic.affermationsapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.recyclerview.widget.RecyclerView
import com.kenetic.affermationsapp.adapter.ItemAdapter
import com.kenetic.affermationsapp.data.DataSource

class MainActivity : AppCompatActivity() {

    //lateinit var binding : ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val myDataSet = DataSource().loadAffirmations()
        //stores list of quotes

        val recyclerView = findViewById<RecyclerView>(R.id.recyler_view)
        //stores recycler view

        recyclerView.adapter = ItemAdapter(this, myDataSet)
        //setting up an adapter
        recyclerView.setHasFixedSize(true)

    }
}